migrations/versions/e553b195fe90_.py

```diff
@@ -10,124 +10,131 @@
 from sqlalchemy.dialects import postgresql
 
 # revision identifiers, used by Alembic.
-revision = 'e553b195fe90'
```

Comment:
~~Why has Black chosen now to reformat this historical migration file?  Or is there an actual change to it?~~

OK spotted it - we're adding the name of the unique constraint. Does Alembic really need that before we can drop it in the new migration? Seems like it should be happy to run the new one without this in the history?

Reply:
If you have the naming convention and then reset the test db, it will re-run all migrations - and then fail to complete the migrations. So we need to update it for that reason.

Reply:
Have fixed this so that black hasn't reformatted the whole file now.